### ISSUE TITLE
How to configure dim/rank for dreambooth lora training to get a larger lora model

### ISSUE THREAD

**Issue by THULiusj** [huggingface/diffusers#2711]
I tried training_dreambooth_lora.py, and the fine-tuned lora model is about 3MB, quite small. How to get a larger Lora model (like 100MB)? How to change the dim rank for lora training, to make the model larger?

Thanks

**Comment by sayakpaul** [comment]
Here you have `LoRAAttnProcessor` that takes care of initializing the LoRA layers in the attention blocks of the UNet: 

https://github.com/huggingface/diffusers/blob/73bdad08a1fae592c73e211b55b83745c487b5ea/examples/dreambooth/train_dreambooth_lora.py#L726

If we look here

https://github.com/huggingface/diffusers/blob/73bdad08a1fae592c73e211b55b83745c487b5ea/src/diffusers/models/attention_processor.py#L356

We notice that `LoRAAttnProcessor` also exposes a `rank` argument. So, you can specify that [here](https://github.com/huggingface/diffusers/blob/73bdad08a1fae592c73e211b55b83745c487b5ea/examples/dreambooth/train_dreambooth_lora.py#L726). 

Cc: @patrickvonplaten

**Comment by THULiusj** [comment]
Got it! Thanks! And to enable text encoder training, is it to set         text_encoder.requires_grad_(True) in training_dreambooth_lora.py

**Comment by sayakpaul** [comment]
Yes, that is correct. Additional changes might be necessary to facilitate its training. Refer to [this script](https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth.py) for a complete example.

**Comment by sayakpaul** [comment]
Closing the issue for now, but feel free to re-open in case needed.
